torch/quantization/_numeric_suite.py

```diff
@@ -242,3 +265,89 @@ def compare_model_stub(float_model, q_model, module_swap_list, data, Logger=Logg
     ob_dict = {}
     get_observer_dict(q_model, ob_dict, Logger)
     return ob_dict
+
+
+def get_matching_activations(float_module, q_module, Logger):
+    r"""Find the matching activation between float and quantized dict.
+
+    Args:
+        float_dict: recording observer dict of the float model
+        quantized_dict: recording observer dict of the quantized model
+
+    Return:
+        act_dict: dict with key corresponding to quantized module names and each
+        entry being a dictionary with two keys 'float' and 'quantized', containing
+        the matching float and quantized activations
+    """
+    float_dict = {}
+    quantized_dict = {}
+    get_observer_dict(q_module, quantized_dict, Logger)
+    get_observer_dict(float_module, float_dict, Logger)
+    act_dict = {}
+    for key in quantized_dict:
+        match_key = _find_match(sorted(float_dict, reverse=True), key, "stats")
+        if match_key is not None:
+            act_dict[key] = {}
+            act_dict[key]["float"] = float_dict[match_key]["tensor_val"]
+            act_dict[key]["quantized"] = quantized_dict[key]["tensor_val"]
+    return act_dict
+
+
+def remove_qconfig(module):
```

Comment:
We need do this as part of convert, please add this logic there.

Reply:
Seems if I remove qconfig in convert(), some tests related to QAT and fusion would fail: 

>   FAIL: 7
    caffe2/test:quantization - test_activations (quantization.test_quantize.TestQuantizationAwareTraining)
    caffe2/test:quantization - test_fuse_module_train (quantization.test_quantize.TestFusion)
    caffe2/test:quantization - test_conv_linear (quantization.test_quantize.TestQuantizationAwareTraining)
    caffe2/test:quantization - test_manual (quantization.test_quantize.TestQuantizationAwareTraining)
    caffe2/test:quantization - test_train_save_load_eval (quantization.test_quantize.TestQuantizationAwareTraining)
    caffe2/test:quantization - test_fusion_sequential_model_train (quantization.test_quantize.TestFusion)
    caffe2/test:quantization - test_compare_model_outputs (quantization.test_numeric_suite.TestEagerModeNumericSuite)

>

Reply:
I see, the problem is occuring because convert is called as part of prepare_qat. Please do the following:
1. Add remove_qconfig as an internal function _remove_qconfig in quantize.py
2. Please add a call to _remove_qconfig in quantize, quantize_qat and quantize_dynamic functions